pyci/rdm/algorithms.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np 
+from abc import ABCMeta, abstractmethod
+class Projection(metaclass=ABCMeta):
+    
+    @abstractmethod
+    def __init__(self, initial_guess,constraints):
+        self.initial_guess = initial_guess
+        self.constraints = constraints
+    
+    @abstractmethod 
+    def optimize(self):
+        pass
+
+class Dykstra(metaclass=ABCMeta):
```

Comment:
you need to inherit the abstract class that is defined above
So, it should be `Dykstra(Projector)`